app/experimenter/experiments/api/v5/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+from django.views.decorators.csrf import csrf_exempt
+from graphene_django.views import GraphQLView
+
+urlpatterns = [
+    url(r"graphql", csrf_exempt(GraphQLView.as_view(graphiql=True)), name="graphql"),
```

Comment:
Ty 🙏

Reply:
Oh name should be qualified with the app as well since it's a single global namespace, so something like

`nimbus-api-graphql`

or wahtever